README.md

```diff
@@ -29,7 +29,7 @@ For example, if you made are making an etcd cluster to be used by a service call
 
     senza create https://raw.github.com/zalando/stups-etcd-cluster/master/etcd-cluster.yaml releaseetcd \
                                    HostedZone=elephant.example.org \
-                                   DockerImage=registry.opensource.zalan.do/acid/etcd-cluster:2.3.1-p10 \
+                                   DockerImage=registry.opensource.zalan.do/acid/etcd-cluster:2.3.6-p11 \
```

Comment:
Previous one was p10, I thought you were in favour of ever increasing patcnumbers right? Else I'll rename to p1